doc/contributing/docs/markup/code.rst

```diff
@@ -41,15 +86,52 @@ Both options produce the same output:
 
 ..  include:: _includes/inline-code.rst
 
+
+Notes on using inline-code
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*   If you have expressions such as ``id==4``, you should format the whole
+    expression as code inline. Also,you can use the words 'equals',
+    'doesn't equal' or other similar words without formatting expression
+    as code. Both variants are correct.
+
+*   Inline code can be used to highlight expressions that are hard to read,
+    for example, words containing ``il``, ``Il`` or ``O0``.
+
+
 Highlighting variables in code
 ------------------------------
 
-If you need to highlight variables in code inline, use the ``:samp:`` role,
-like this:
+If you need to mark up a placeholder inside code inline, use the ``:samp:``
+role, like this:
 
 ..  literalinclude:: _includes/samp.rst
     :language: rst
 
 And you will get this:
 
-..  include:: _includes/samp.rst
\ No newline at end of file
+..  include:: _includes/samp.rst
+
+If you need to highlight a replaceable code in the code-block, use
+the following syntax:
+
+..  literalinclude:: _includes/highlight.rst
+    :language: rst
+
+The output will look like this:
+
+.. include:: _includes/highlight.rst
+
+Highlighting file name or directory in code
```

Comment:
```suggestion
Highlighting file or directory name in code
```

But it's not a file name in code. It's a standalone file name, probably with a placeholder. So,

```suggestion
Formatting file and directory names
```